### PR TITLE
loccheckin

### PR DESCRIPTION
**Pull request by cristianosuzuki77**
Latest loc string and fixes for the following translation tickets:
https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1163530
https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1180932
https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1178477
https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1180932
https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1174024

This PR replaces https://github.com/dotnet/roslyn/pull/47474 with an additional fix (1174024)

### CODE REVIEW

File: src/Analyzers/CSharp/Analyzers/xlf/CSharpAnalyzersResources.de.xlf [hunk 1]
```diff
@@ -264,7 +264,7 @@
       </trans-unit>
       <trans-unit id="typeof_can_be_converted_ to_nameof">
         <source>'typeof' can be converted  to 'nameof'</source>
-        <target state="new">'typeof' can be converted  to 'nameof'</target>
+        <target state="translated">"typeof" kann in "nameof" konvertiert werden.</target>
```
Comment (JoeRobich) [inline src/Analyzers/CSharp/Analyzers/xlf/CSharpAnalyzersResources.de.xlf:267]: Original string did not include a `.`

```suggestion
        <target state="translated">"typeof" kann in "nameof" konvertiert werden</target>
```

File: src/Analyzers/CSharp/Analyzers/xlf/CSharpAnalyzersResources.ko.xlf [hunk 2]
```diff
@@ -264,7 +264,7 @@
       </trans-unit>
       <trans-unit id="typeof_can_be_converted_ to_nameof">
         <source>'typeof' can be converted  to 'nameof'</source>
-        <target state="new">'typeof' can be converted  to 'nameof'</target>
+        <target state="translated">'typeof'를 'nameof'로 변환할 수 있습니다.</target>
```
Comment (JoeRobich) [inline src/Analyzers/CSharp/Analyzers/xlf/CSharpAnalyzersResources.ko.xlf:267]: Original string did not include a `.`

```suggestion
        <target state="translated">'typeof'를 'nameof'로 변환할 수 있습니다</target>
```

File: src/Analyzers/CSharp/Analyzers/xlf/CSharpAnalyzersResources.ru.xlf [hunk 3]
```diff
@@ -264,7 +264,7 @@
       </trans-unit>
       <trans-unit id="typeof_can_be_converted_ to_nameof">
         <source>'typeof' can be converted  to 'nameof'</source>
-        <target state="new">'typeof' can be converted  to 'nameof'</target>
+        <target state="translated">"typeof" можно преобразовать в "nameof".</target>
```
Comment (JoeRobich) [inline src/Analyzers/CSharp/Analyzers/xlf/CSharpAnalyzersResources.ru.xlf:267]: Original string did not include a `.`
```suggestion
        <target state="translated">"typeof" можно преобразовать в "nameof"</target>
```

File: src/Analyzers/VisualBasic/Analyzers/xlf/VisualBasicAnalyzersResources.de.xlf [hunk 4]
```diff
@@ -4,7 +4,7 @@
     <body>
       <trans-unit id="GetType_can_be_converted_to_NameOf">
         <source>'GetType' can be converted to 'NameOf'</source>
-        <target state="new">'GetType' can be converted to 'NameOf'</target>
+        <target state="translated">"GetType" kann in "NameOf" konvertiert werden.</target>
```
Comment (JoeRobich) [inline src/Analyzers/VisualBasic/Analyzers/xlf/VisualBasicAnalyzersResources.de.xlf:7]: Original string did not include a `.`

```suggestion
        <target state="translated">"GetType" kann in "NameOf" konvertiert werden</target>
```

File: src/Analyzers/VisualBasic/Analyzers/xlf/VisualBasicAnalyzersResources.ko.xlf [hunk 5]
```diff
@@ -4,7 +4,7 @@
     <body>
       <trans-unit id="GetType_can_be_converted_to_NameOf">
         <source>'GetType' can be converted to 'NameOf'</source>
-        <target state="new">'GetType' can be converted to 'NameOf'</target>
+        <target state="translated">'GetType'을 'NameOf'로 변환할 수 없습니다.</target>
```
Comment (JoeRobich) [inline src/Analyzers/VisualBasic/Analyzers/xlf/VisualBasicAnalyzersResources.ko.xlf:7]: Original string did not include a `.`

```suggestion
        <target state="translated">'GetType'을 'NameOf'로 변환할 수 없습니다</target>
```

File: src/Analyzers/VisualBasic/Analyzers/xlf/VisualBasicAnalyzersResources.ru.xlf [hunk 6]
```diff
@@ -4,7 +4,7 @@
     <body>
       <trans-unit id="GetType_can_be_converted_to_NameOf">
         <source>'GetType' can be converted to 'NameOf'</source>
-        <target state="new">'GetType' can be converted to 'NameOf'</target>
+        <target state="translated">"GetType" можно преобразовать в "NameOf".</target>
```
Comment (JoeRobich) [inline src/Analyzers/VisualBasic/Analyzers/xlf/VisualBasicAnalyzersResources.ru.xlf:7]: Original string did not include a `.`

```suggestion
        <target state="translated">"GetType" можно преобразовать в "NameOf"</target>
```

File: src/Compilers/CSharp/Portable/xlf/CSharpResources.cs.xlf [hunk 7]
```diff
@@ -29,12 +29,12 @@
       </trans-unit>
       <trans-unit id="ERR_AddressOfMethodGroupInExpressionTree">
         <source>'&amp;' on method groups cannot be used in expression trees</source>
-        <target state="new">'&amp;' on method groups cannot be used in expression trees</target>
+        <target state="translated">&amp; pro skupiny metod se nedá použít ve stromech výrazů.</target>
```
Comment (JoeRobich) [inline src/Compilers/CSharp/Portable/xlf/CSharpResources.cs.xlf:32]: Original string did not include a `.`

```suggestion
        <target state="translated">&amp; pro skupiny metod se nedá použít ve stromech výrazů</target>
```

File: src/Compilers/CSharp/Portable/xlf/CSharpResources.cs.xlf [hunk 8]
```diff
@@ -44,7 +44,7 @@
       </trans-unit>
       <trans-unit id="ERR_AmbigBinaryOpsOnDefault">
         <source>Operator '{0}' is ambiguous on operands '{1}' and '{2}'</source>
-        <target state="new">Operator '{0}' is ambiguous on operands '{1}' and '{2}'</target>
+        <target state="translated">Operátor {0} je na operandech {1} a {2} nejednoznačný.</target>
```
Comment (JoeRobich) [inline src/Compilers/CSharp/Portable/xlf/CSharpResources.cs.xlf:47]: Original string did not include a `.`

```suggestion
        <target state="translated">Operátor {0} je na operandech {1} a {2} nejednoznačný</target>
```

File: src/Compilers/CSharp/Portable/xlf/CSharpResources.cs.xlf [hunk 9]
```diff
@@ -89,12 +89,12 @@
       </trans-unit>
       <trans-unit id="ERR_AwaitForEachMissingMember">
         <source>Asynchronous foreach statement cannot operate on variables of type '{0}' because '{0}' does not contain a suitable public instance or extension definition for '{1}'</source>
-        <target state="needs-review-translation">Asynchronní příkaz foreach nejde použít pro proměnné typu {0}, protože {0} neobsahuje vhodnou veřejnou definici instance pro {1}.</target>
+        <target state="translated">Asynchronní příkaz foreach nejde použít pro proměnné typu {0}, protože {0} neobsahuje vhodnou veřejnou definici instance nebo rozšíření pro {1}.</target>
```
Comment (JoeRobich) [inline src/Compilers/CSharp/Portable/xlf/CSharpResources.cs.xlf:92]: Original string did not include a `.`

```suggestion
        <target state="translated">Asynchronní příkaz foreach nejde použít pro proměnné typu {0}, protože {0} neobsahuje vhodnou veřejnou definici instance nebo rozšíření pro {1}</target>
```

File: src/Compilers/CSharp/Portable/xlf/CSharpResources.cs.xlf [hunk 10]
```diff
@@ -104,22 +104,22 @@
...
       <trans-unit id="ERR_BadInheritanceFromRecord">
         <source>Only records may inherit from records.</source>
-        <target state="new">Only records may inherit from records.</target>
+        <target state="translated">Ze záznamů můžou dědit jenom záznamy.</target>
         <note />
       </trans-unit>
       <trans-unit id="ERR_BadInitAccessor">
         <source>The 'init' accessor is not valid on static members</source>
-        <target state="new">The 'init' accessor is not valid on static members</target>
+        <target state="translated">Přístupový objekt init není platný pro statické členy.</target>
```
Comment (JoeRobich) [inline src/Compilers/CSharp/Portable/xlf/CSharpResources.cs.xlf:122]: Original string did not include a `.`

```suggestion
        <target state="translated">Přístupový objekt init není platný pro statické členy</target>
```

**Review by JoeRobich — CHANGES_REQUESTED** [review]
@cristianosuzuki77 & @wli3 I didn't work through the entire set of string changes, but many translated strings are adding `.` when they are not present in the source string. Is this expected?

**Comment by wli3** [comment]
@cristianosuzuki77?

**Comment by cristianosuzuki77** [comment]
Checking with translators.

**Comment by wli3** [comment]
@cristianosuzuki77 ping. Any update?

**Comment by cristianosuzuki77** [comment]
The translators confirmed instances of where the dots were introduced are not correct. A new pull request will be created.
Root cause analysis as well is undergo.

**Comment by wli3** [comment]
@cristianosuzuki77 where is the new PR? It has been several weeks for getting back and forth for this issue, is there an ETA? The VS loc deadline is coming soon.

**Comment by cristianosuzuki77** [comment]
Should be in tomorrow.

**Comment by cristianosuzuki77** [comment]
Today's PR is still showing up few spots with the extra dot. Translation team says it is expected, but I am asking them to clarify before proceeding. ETA is pushed to tomorrow.

**Comment by cristianosuzuki77** [comment]
Addressing this via https://github.com/dotnet/roslyn/pull/48037

**Pull request closed without merging**